src/components/Comments/WriteComment.js

```diff
@@ -244,14 +253,10 @@ export const styles = StyleSheet.create({
     justifyContent: 'center'
   },
   text: {
-    flex: 1,
-    fontSize: 12,
+    fontSize: 14,
     fontFamily: fonts.inter[400],
     color: COLORS.black,
-    backgroundColor: COLORS.gray110,
-    maxHeight: 100,
-    paddingTop: Platform.OS === 'ios' ? 10 : 5,
-    paddingBottom: Platform.OS === 'ios' ? 10 : 5
+    maxHeight: 100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the text has been adjusted. The `flex: 1` property has been removed, and the font size has been increased. Make sure these changes align with the desired design and do not cause any issues in different screen sizes or orientations.


<!-- This is an auto-generated comment by OSS CodeRabbit -->